src/components/Voting.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { Vote } from '../../api-patch';
+
+export const Voting = ({ articleId, initialVoteCount }) => {
+
+    const [voteCount, setVoteCount] = useState(Number(initialVoteCount));
+
+
+    const handleVote = (voteChange) => {
+        setVoteCount((prevVoteCount) => prevVoteCount + voteChange);
+
+        Vote(articleId, { inc_votes: voteChange,
+            article_id: articleId
+        })
+            .then((response) => {
+                window.location.reload();                
```

Comment:
we dont need to (or want to) reload the page here as that's why we are optimistically rendering the vote change